### PR TITLE
updated iris recipe to v2.2

### PR DESCRIPTION
**Pull request by corinnebosley**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by corinnebosley** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ requirements:
     - cartopy >=0.14
     - cf_units >=2
     - cftime
-    - dask >=0.17.1
-    - numpy
+    - dask
+    - numpy >=1.14, !=1.15.2
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Why not 1.15.2? If you are worried about the py37 issue I think we have the backport patch already.

Comment (corinnebosley) [reply recipe/meta.yaml]: What py37 issue?  We were having problems with build number 1 of numpy 1.15.2.  Is that related?

Comment (dkillick) [reply recipe/meta.yaml]: @ocefpaf see https://travis-ci.org/SciTools/iris/jobs/439687656 for an example of the build problem seemingly being caused by NumPy v1.15.2 build 1

**Comment by dkillick** [comment]
@ocefpaf we're keen to get Iris v2.2 broadly available so I'm going to merge this now given that all the checks are happy. Not sure Iris is quite ready for Py3.7 yet (but we may well target that for the next Iris release) and I'm happy with the weird NumPy pin in this build. We can always add another build if we need to!

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml]: > What py37 issue? 

This one: https://github.com/numpy/numpy/pull/12062

> @ocefpaf see travis-ci.org/SciTools/iris/jobs/439687656 for an example of the build problem seemingly being caused by NumPy v1.15.2 build 1

Thanks @dkillick! That was fixed as well. Can you retry the CIs and test again?

**Pull request merged**